cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/efs/AwsEfsFileSystem.java

```diff
@@ -0,0 +1,188 @@
+package com.sequenceiq.cloudbreak.cloud.aws.efs;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.sequenceiq.cloudbreak.cloud.model.SpiFileSystem;
+import com.sequenceiq.common.model.FileSystemType;
+
+public class AwsEfsFileSystem {
+    // TODO: move these keys to a file under cloud-api
+    public static final String KEY_BACKUP_POLICY_STATUS = "backupPolicyStatus";
+
+    public static final String KEY_ENCRYPTED = "encrypted";
+
+    public static final String KEY_FILESYSTEM_POLICY = "fileSystemPolicy";
+
+    public static final String KEY_FILESYSTEM_TAGS = "fileSystemTags";
+
+    public static final String KEY_FILESYSTEM_TAGS_NAME = "Name";
+
+    public static final String KEY_KMSKEYID = "kmsKeyId";
+
+    public static final String KEY_LIFECYCLE_POLICIES = "lifeCyclePolicies";
+
+    public static final String KEY_PERFORMANCE_MODE = "performanceMode";
+
+    public static final String KEY_PROVISIONED_THROUGHPUT_INMIBPS = "provisionedThroughputInMibps";
+
+    public static final String KEY_THROUGHPUT_MODE = "throughputMode";
+
+    // Allowed values: DISABLED | DISABLING | ENABLED | ENABLING
+    private static final String BACKUP_POLICY_STATUS_DEFAULT = "DISABLED";
+
+    // Allowed values: generalPurpose | maxIO
+    private static final String PERFORMANCE_MODE_DEFAULT = "generalPurpose";
+
+    // Allowed values: bursting | provisioned
+    private static final String THROUGHPUT_MODE_BURSTING = "bursting";
```

Comment:
Is there a reason these were done as strings and not enums?

Reply:
this is used directly in CloudFormation template. I can try if it can handle enum correctly

Reply:
I have used enum now.